spec/features/card_trainer_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+require "helpers"
+
+describe "card training process", type: :feature do 
+  before :each do
+
+  	card = create(:card)
```

Comment:
Useless assignment to variable - `card`.<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.